packages/react-ui/src/components/atoms/Typography.d.ts

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { TypographyTypeMap } from '@mui/material/Typography';
+import { OverridableComponent, OverrideProps } from '@mui/material/OverridableComponent';
+
+export type CartoTypographyTypeMap<D extends React.ElementType = 'span'> =
+  TypographyTypeMap<
+    {
+      /**
+       * Font weight for Carto typography:
+       *
+       *  * `'regular'` - 400
+       *  * `'medium'` - 500
+       *  * `'strong'` - 600
+       */
+      weight?: CartoFontWeight;
+      italic?: boolean;
+    },
+    D
+  >;
+
+export type CartoFontWeight = 'regular' | 'medium' | 'strong';
```

Comment:
I understand this is pure 'CARTO', right? No option to extend or reuse any related typing from MUI5... correct?

Reply:
Yes, according to implementation this is purely our concept.

Reply:
yes, it is pure CARTO DS definition.